mollie/api/client.py

```diff
@@ -295,6 +265,10 @@ def setup_oauth_authorization_response(self, authorization_response):
         self.set_token(self.access_token)
         return self.access_token
 
+    # TODO Implement https://docs.mollie.com/reference/oauth2/revoke-token
```

Comment:
Is this TODO intentional?

Reply:
Yes, but the change is not in scope for this PR (or a blocker for an upcoming release).